OpenTESArena/src/Interface/CinematicPanel.h

```diff
@@ -22,8 +22,8 @@ class CinematicPanel : public Panel
 	double secondsPerImage, currentSeconds;
 	int imageIndex;
 public:
-	CinematicPanel(Game *game, const std::string &paletteName,
-		const std::string &sequenceName, double secondsPerImage,
+	CinematicPanel(Game *game, const std::string &sequenceName,
+		const std::string &paletteName, double secondsPerImage,
```

Comment:
I don't know why it works. But I changed it so that `paletteName` is always first.